templates/greenlight/env.j2

```diff
@@ -231,4 +231,4 @@ DB_PASSWORD={{ bbb_greenlight_db_password }}
 #   open - For open registration
 #   invite - For invite only registration
 #   approval - For approve/decline registration
-DEFAULT_REGISTRATION=open
```

Comment:
Thanks so far! Could you mention this variable in the README, so it is documented somewhere?

Reply:
Done in https://github.com/n0emis/ansible-role-bigbluebutton/pull/3/commits/90e51b3f9b8401dd5d5c97da6f96fcb51267f384